pkg/encryption/store.go

```diff
@@ -34,6 +34,11 @@ import (
 type Store struct {
 	roots      map[string]*node
 	defaultKey *storj.Key
+
+	// TODO: The below field makes it so we can interoperate with
+	// the network without having encryption keys. paths will be encrypted but
+	// base64-encoded, and certain metadata will be unable to be retrieved.
+	EncryptionBypass bool
```

Comment:
Why is this a TODO?

Reply:
I'm not sure. I could make it an `NB:` or just remove the `TODO:`.

Reply:
Yeah, perhaps just rephrase it so it becomes the description for this field, starting with "EncryptionBypass ... "

Reply:
:+1: